scripts/ideogram/matrix_to_ideogram_annots.py

```diff
@@ -223,8 +232,8 @@ def compute_gene_expression_means(self, matrix, cluster_group, scope, cluster_na
 
         cells = matrix['cells']
 
-        print('cells')
-        print(cells)
+        # print('cells')
```

Comment:
Minor nit - I'd remove these if you've already commented them out.

Reply:
Done!